homework7.playground/Contents.swift

```diff
@@ -0,0 +1,44 @@
+import UIKit
+
+class School {
+    var students = [Student]()
+    
+    init(students: [Student] = [firstStudent, secondStudent, thirdStudent, fourthStudent]) {
+        self.students = students
+    }
+    func printInfo() {
+        let info = students
+        print(info)
+    }
+}
+
+// let school = School()
```

Comment:
You need to pass `name` and `surname` because they're set in your init and don't have a default value like `marks` and `grade`